idaes/models/properties/general_helmholtz/helmholtz_functions.py

```diff
@@ -769,7 +803,9 @@ def w_vap(self, **kwargs):
         )
 
     def viscosity(self, **kwargs):
-        """Mixed phase Helmholtz free energy"""
+        """Mixed phase viscosity (value for two-phase is not meaningful)
+        use the liquid or vapor version if two phases are expected.
+        """
```

Comment:
This is a little weird, but if you don't know if you will have a liquid or a vapor, but don't expect two-phase it can be useful.  It also lets you go into the two-phase region while solving without running into a zero derivative.  It could be useful numerically, so I included it.